node-graph/gcore/src/graphic_element/renderer.rs

```diff
@@ -20,22 +20,43 @@ use std::fmt::Write;
 #[cfg(feature = "vello")]
 use vello::*;
 
+#[derive(Clone, Debug, PartialEq, serde::Serialize, serde::Deserialize)]
+pub enum ClickTargetGroup {
+	Subpath(bezier_rs::Subpath<PointId>),
+	PointGroup(ManipulatorGroup<PointId>),
```

Comment:
I don't really understand why this data structure is used anyway? An unconnected point won't have any handles. It would be far more ergonomic just to use a DVec2 and a PointId.

Reply:
If you do this, we have the additional confusion of a `ClickTargetGroup` being a single point and not a group.